### ISSUE TITLE
Unify text sizing for missing fonts

### ISSUE THREAD

**Issue by PavelHolec** [kiwicom/orbit-swiftui#185]
When custom fonts are missing, the non-formatted part of Text component has a wrong size for accessibility sizes.

Probably related to

`return .custom(name, size: size, relativeTo: style)`
